metadb/rqlite/tcp/mux_test.go

```diff
@@ -41,7 +41,8 @@ func TestMux(t *testing.T) {
 		}
 		mux.Timeout = 200 * time.Millisecond
 		if !testing.Verbose() {
-			mux.Logger = log.New(ioutil.Discard, "", 0)
+			mux.Logger = log.DefaultLogger.WithField("prefix", "mux_test")
```

Comment:
It's really weird to use standard log in unit test. I understand, it's not your idea. but it is better to implement it in a more correct way, using `t.Logf` If we have to change the code anyway

Reply:
done